deathbeds/jupyak
# try lite vs 4.1.0 etc

## what kind of request is this?

- [x] a preview site on ReadTheDocs
  - [x] it only adds a `jupyak_config.toml`
- [ ] a `jupyak` code change
  - [ ] it adds new features, described below

## for preview sites

### how should a visitor use the preview site?

1. go to https://jupyak--21.org.readthedocs.build/en/21/_static/work/lite/lab/
2. navigate to _Help &raquo; Licenses_
3. open the big _JupyterLite_ bundle
4. see `4.1.0.alpha.4` (or later) versions for `@jupyterlab` components

### where is the main discussion?

- https://github.com/jupyterlite/jupyterlite/pull/1263
